tests/e2e.sh

```diff
@@ -55,6 +57,7 @@ deploy_operator() {
     $KUBECTL apply -f environments/dev/manifests/minio-service.yaml
     $KUBECTL apply -f deploy/crds
     $KUBECTL apply -f deploy/
+    $KUBECTL patch deployment observatorium-operator --patch '{"spec": {"template": {"spec": {"containers": [{"name": "observatorium-operator","image": "obs-operator:test"}]}}}}'
```

Comment:
if we directly build and load `quay.io/observatorium/observatorium-operator:latest` then we don't need to patch anything here

Reply:
We should not use `latest` according to this: https://kind.sigs.k8s.io/docs/user/quick-start/#loading-an-image-into-your-cluster.
But for some reasons it worked...